charts/kong/README.md

```diff
@@ -686,6 +686,8 @@ Kong is going to be deployed.
 
 #### Kong Enterprise Docker registry access
```

Comment:
```suggestion
#### Kong Enterprise Docker registry access (only for Kong <2.3)
```

Reply:
You English is better than mine  :)

Reply:
Probably not :slightly_smiling_face:  - rewording comments are just about leaving no doubt about the corner cases.

This particular comment makes it evident from the header of the section that for modern Kong versions it's ok to gloss over it.